<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mRTCClient</class>
 <widget class="QWidget" name="mRTCClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="buttom_status" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>796</y>
     <width>1417</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightblue;</string>
   </property>
   <widget class="QLabel" name="status_tip1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="status_tip2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="status_tip3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="middle_setting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>1181</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>设置:</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="middle_groupBox_1">
      <property name="title">
       <string>音视频设备:</string>
      </property>
      <widget class="QLabel" name="label_cam">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>49</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>摄像头:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cam_devices">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>40</y>
         <width>211</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_mic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>49</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>麦克风:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="mic_devices">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>80</y>
         <width>211</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_devices">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>151</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>启动音视频设备</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_preview">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>164</y>
         <width>151</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>开始本地预览</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="middle_groupBox_2">
      <property name="title">
       <string>推流: </string>
      </property>
      <widget class="QLabel" name="label_server_url">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>49</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_server_url">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>40</y>
         <width>271</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>www.127.0.0.1:8080</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_uid">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>49</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>用户ID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_uid">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>80</y>
         <width>271</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_streamid">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>34</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>流ID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_streamid">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>120</y>
         <width>271</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start_push">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>211</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>开始推流</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="middle_groupBox_3">
      <property name="title">
       <string>拉流: </string>
      </property>
      <widget class="QLabel" name="label_pull_streamid_1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>28</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>流1:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pull_streamid_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>28</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>流2:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_pull_stream_1">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>50</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_pull_stream_2">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>100</y>
         <width>116</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start_pull_1">
       <property name="geometry">
        <rect>
         <x>186</x>
         <y>50</y>
         <width>131</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>开始拉流</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start_pull_2">
       <property name="geometry">
        <rect>
         <x>182</x>
         <y>100</y>
         <width>131</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>开始拉流</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pull_streamid_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>28</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>流3:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start_pull_3">
       <property name="geometry">
        <rect>
         <x>186</x>
         <y>150</y>
         <width>131</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>开始拉流</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_pull_stream_3">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>150</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="top_video" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1171</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
